### PR TITLE
LOOK-218 Added new seeds with types description

### PR DESCRIPTION
**Pull request by Simarglok**

### CODE REVIEW

File: seeds/order_discount_type.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+order_discount_id,order_discount_name
```
Comment (iplotnikov-telemedicine) [inline seeds/order_discount_type.csv:1]: let's rename the columns into discount_id and discount_name respectively

File: seeds/delivery_status.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+delivery_status,delivery_status_name
+STATUS_NONE,none
+STATUS_PENDING,pending
+STATUS_DRIVER_ON_ROUTE,driver_on_route
+STATUS_DRIVER_IS_ASSIGNED,driver_is_assigned
+STATUS_NOT_HOME,not_home
+STATUS_DELIVERY_CANCELED,delivery_canceled
+STATUS_DELIVERED,delivered
+STATUS_SHIPPED,shipped
```
Comment (iplotnikov-telemedicine) [inline seeds/delivery_status.csv:9]: what about removing this seed, looks like redundant stuff

File: seeds/order_discount_type.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+1,ORDER_ITEM_CUSTOM_DISCOUNT_TYPE
+2,ORDER_SIMPLE_DISCOUNT_TYPE
+3,ORDER_CUSTOM_DISCOUNT_TYPE
+5,ORDER_BONUS_POINT_DISCOUNT_TYPE
+6,ORDER_ITEM_SIMPLE_DISCOUNT_TYPE
```
Comment (iplotnikov-telemedicine) [inline seeds/order_discount_type.csv:6]: let's name the seed as discount_types as they store both order and item discounts in the same table

**Review by iplotnikov-telemedicine — CHANGES_REQUESTED** [review]
let's rename all the seeds using plural nouns e.g. "discount_types", "event_types" just to stick to the existing naming approach

File: seeds/order_discount_type.csv (hunk 1 above)
Comment (Simarglok) [reply seeds/order_discount_type.csv:1]: done

File: seeds/delivery_status.csv (hunk 2 above)
Comment (Simarglok) [reply seeds/delivery_status.csv:9]: done

File: seeds/order_discount_type.csv (hunk 3 above)
Comment (Simarglok) [reply seeds/order_discount_type.csv:6]: done

**Comment by Simarglok** [comment]
Fixed at https://github.com/iplotnikov-telemedicine/indica/pull/6

**Pull request merged**